docs/development/testing.md

```diff
@@ -154,6 +154,23 @@ an `HTTP 400 error`.
 In that case, configure an alternate load method by referencing the name of the test BPMN diagram in the load configuration
 settings.
 
+### Performance tests
+For now these tests are defined under `/test/e2e/performance/` as the performance is being measured for complex(e2e) tasks:
+- display diagram
+- zoom in / zoom out loaded diagram
+
+#### Why?
+To have visibility if new changes have impact on current performance
+
+#### Results
+The results are under `/test/e2e/performance/data/`
+Preview is available in the browser, simply open file `/bpmn-visu-js/test/e2e/performance/index.html` in your favourite browser
+
+#### Run the performance tests
```

Comment:
When I run the test on MacOS, I have only this in the file **test/e2e/performance/data/macos/data.js**: 
`const data = { "load": [], "zoom": [] }`

Reply:
```shell
➜  e2e git:(tests-performance_measurement) jest --runInBand --detectOpenHandles --testNamePattern=^performance
Determining test suites to run...
DevTools listening on ws://127.0.0.1:52262/devtools/browser/2c2c5705-122d-464f-8414-fc6d3e797b53

Jest dev-server output:
[Jest Dev server] 
> bpmn-visualization@0.6.0 start /Users/csouchet/Source/Process Analytics/BPMN-Js
> rollup -c --silent --environment devMode:true "--config-server-port" "10002"

[Jest Dev server] src/bpmn-visualization.ts:
app - 188.45 KB (70.97%)
fast-xml-parser - 42.41 KB (15.97%)
entities - 33.1 KB (12.47%)
rollup helpers - 1.57 KB (0.59%)
http://localhost:10002 -> /Users/csouchet/Source/Process Analytics/BPMN-Js/dist

Test Suites: 8 skipped, 0 of 8 total
Tests:       78 skipped, 78 total
Snapshots:   0 total
Time:        8.201 s
Ran all test suites with tests matching "^performance".

```

Reply:
it looks like it skipped all of the tests
perhaps there is an issue ith '^' character

Reply:
it supposed to be OK now